frontend/src/lib/components/PropertyFilters/propertyFilterLogic.js

```diff
@@ -0,0 +1,84 @@
+import { kea } from 'kea'
+import api from '../../../lib/api'
+import { userLogic } from 'scenes/userLogic'
+
+export const propertyFilterLogic = kea({
+    key: props => props.onChange,
+    connect: {
+        values: [userLogic, ['eventProperties']],
+    },
+    actions: () => ({
+        loadEventProperties: true,
+        setProperties: properties => ({ properties }),
+        update: filters => ({ filters }),
+        setInitial: filters => ({ filters }),
+        set: (index, key, value) => ({ index, key, value }),
+        newFilter: true,
+        remove: index => ({ index }),
+    }),
+
+    loaders: ({ values }) => ({
+        properties: {
+            loadPeopleProperties: async () => {
+                return await api.get('api/person/properties')
+            },
+        },
+    }),
+
+    reducers: ({ actions, values }) => ({
+        properties: [
+            [],
+            {
+                [actions.loadPeoplePropertiesSuccess]: (_, { properties }) =>
+                    properties.map(property => ({
+                        label: property.name,
+                        value: property.name,
+                    })),
```

Comment:
why not move this into the loader?

Reply:
![image](https://user-images.githubusercontent.com/1727427/79342512-2507b100-7f25-11ea-80e8-dc31d1db46e0.png)

Like this? I couldn't get this to work but I'm sure I'm missing something?

Reply:
Yeah.. this should do the trick? I'm not sure what's not working here. You can use the redux devtools extension to [debug it](https://kea-docs.netlify.com/docs/debugging) further if needed.

Reply:
As discussed, will leave as is